src/Tokenizer/Transformer/BraceClassInstantiationTransformer.php

```diff
@@ -62,18 +62,19 @@ public function process(Tokens $tokens, Token $token, $index)
         }
```

Comment:
oops, sorry @julienfalque didn't meant to push directly to your branch, but I don't want to force push it out again... feel free to revert!

Reply:
PHP 5.6 fails because my test code is to modern, PHP 7.1 is broken pipe... so all good... well almost ;)